storybook/src/pages/amsterdam/ProductPage/ProductPage.tsx

```diff
@@ -0,0 +1,278 @@
+          Betalen
+        </Heading>
+        <Paragraph className="ams-mb--md">
+          Asperiores autem culpa, cum deserunt distinctio dolores dolorum exercitationem labore libero mollitia non
+          obcaecati pariatur qui quisquam reiciendis repellat ullam.{' '}
+        </Paragraph>
+        <Button className="ams-mb--xl" variant="primary">
+          Betalen
+        </Button>
+        <Heading className="ams-mb--sm" id="zo-lang-duurt-het" level={2}>
+          Zo lang duurt het
+        </Heading>
+        <Paragraph className="ams-mb--xl">
+          Blanditiis ea ipsum maiores optio quo sapiente, unde ut. Ducimus eius eum ex impedit ipsam ipsum?{' '}
+        </Paragraph>
+        <Heading className="ams-mb--sm" id="bijzonderheden" level={2}>
+          Bijzonderheden
+        </Heading>
+        <Paragraph className="ams-mb--xl">
+          Consequatur deleniti dolor ea, error, eum excepturi fuga fugiat fugit harum inventore laboriosam laudantium
+          odio quam quidem quod reprehenderit, saepe sapiente sunt suscipit tenetur vitae voluptate voluptates
+          voluptatum? Dicta dolor eos est harum ipsa ipsum labore rem totam ut vel!
+        </Paragraph>
+
+        <Heading className="ams-mb--sm" id="bijzonderheden" level={2}>
+          Contact
+        </Heading>
+        <Paragraph className="ams-mb--md">
+          Hebt u een vraag en kunt u het antwoord niet vinden? Neem dan contact met ons op via{' '}
+          <Link
+            href="https://formulieren.amsterdam.nl/TriplEforms/DirectRegelen/formulier/nl-NL/evAmsterdam/Contactformulier.aspx/Contactformulier"
+            variant="inline"
+          >
+            het contactformulier
+          </Link>
+          .
+        </Paragraph>
+        <Heading className="ams-mb--sm" id="bijzonderheden" level={3} size="level-5">
+          Hulp nodig bij het aanvragen?
+        </Heading>
+        <Paragraph className="ams-mb--md">
+          Ga naar een{' '}
+          <Link href="https://www.buurtteamamsterdam.nl" variant="inline">
+            buurtteam
+          </Link>
+          .
+        </Paragraph>
+        <Heading className="ams-mb--sm" id="bijzonderheden" level={3} size="level-5">
+          Bellen
+        </Heading>
+        <Paragraph>
+          <Link href="+31612341234">06 1234 1234</Link>
+        </Paragraph>
+        <Paragraph className="ams-mb--xl">Maandag tot en met donderdag van 09.00 - 17.00 uur</Paragraph>
+
+        <Heading className="ams-mb--sm" id="meer-informatie" level={2}>
+          Meer informatie (knw relaties)
+        </Heading>
+        <LinkList>
+          <LinkList.Link href="#">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</LinkList.Link>
+          <LinkList.Link href="#">Aspernatur cupiditate illo officiis provident quae quidem voluptatum?</LinkList.Link>
+        </LinkList>
+      </Grid.Cell>
+
+      <Grid.Cell className="ams-order--narrow-before" span={{ narrow: 4, medium: 2, wide: 3 }}>
```

Comment:
This reordering messes up the tab order. On narrow screens, the TableOfContents is shown before the main content, but in the tab order it is after the main content.

It's conventional to show TOCs on the left side of the screen, which would also solve this issue.